src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/in/Microsoft.Android.Sdk.BundledVersions.in.targets

```diff
@@ -10,12 +10,16 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   <PropertyGroup>
     <AndroidNETSdkVersion>@ANDROID_PACK_VERSION_LONG@</AndroidNETSdkVersion>
     <XamarinAndroidVersion>@ANDROID_PACK_VERSION_LONG@</XamarinAndroidVersion>
+    <_AndroidLatestStableApiLevel>@ANDROID_LATEST_STABLE_API_LEVEL@</_AndroidLatestStableApiLevel>
   </PropertyGroup>
   <PropertyGroup>
     <_AndroidTargetingPackId Condition="$(TargetPlatformVersion.EndsWith('.0'))">$(TargetPlatformVersion.Substring(0, $(TargetPlatformVersion.LastIndexOf('.0'))))</_AndroidTargetingPackId>
     <_AndroidTargetingPackId Condition="'$(_AndroidTargetingPackId)' == ''">$(TargetPlatformVersion)</_AndroidTargetingPackId>
-    <_AndroidRuntimePackId Condition=" '$(_AndroidTargetingPackId)' &lt; '@ANDROID_LATEST_STABLE_API_LEVEL@' ">@ANDROID_LATEST_STABLE_API_LEVEL@</_AndroidRuntimePackId>
+    <!-- NOTE: adjust if a TargetFramework supports multiple API levels -->
+    <_AndroidWarnOnTargetPlatformVersion Condition=" '$(_AndroidTargetingPackId)' &lt; '$(_AndroidLatestStableApiLevel)' ">$(_AndroidTargetingPackId)</_AndroidWarnOnTargetPlatformVersion>
```

Comment:
This should also "do something" when `'$(_AndroidTargetingPackId)' &gt;  '$(_AndroidLatestStableApiLevel)'`.

This should also take `$(EnablePreviewFeatures)` into consideration, so that *when* net8.0-android35.0 previews are available, `$(TargetFramework)`=net8.0-android35 continues to emit an XA1038 error, while `TargetFramework=net8.0-android35;EnablePreviewFeatures=true` succeeds.

Reply:
35.0 gets a different error:
```
(_CheckForInvalidTargetPlatformVersion target) -> 
dotnet/sdk/9.0.100-alpha.1.23628.5/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(243,5):
error NETSDK1140: 35.0 is not a valid TargetPlatformVersion for Android.
Valid versions include:
21.0
22.0
23.0
24.0
25.0
26.0
27.0
28.0
29.0
30.0
31.0
32.0
33.0
34.0
```